packages/app-cli/tests/md_to_html/checkboxes_1.html

```diff
@@ -0,0 +1,73 @@
+			label.classList.remove(this.checked ? 'checkbox-label-unchecked' : 'checkbox-label-checked');
+			label.classList.add(this.checked ? 'checkbox-label-checked' : 'checkbox-label-unchecked');
+		} catch (error) {
+			console.warn('Checkbox unchecked:0 error', error);
+		}
+		return true;
+	"><label id="cb-label-md-checkbox-0" for="md-checkbox-0" class="checkbox-label-unchecked">The order of checkboxes tests matters.</label></div></li>
+<li class="md-checkbox joplin-checkbox"><div class="checkbox-wrapper"><input type="checkbox" id="md-checkbox-1" onclick="
+		try {
+			if (this.checked) {
+				this.setAttribute('checked', 'checked');
+			} else {
+				this.removeAttribute('checked');
+			}
+
+			postMessage('checkboxclick:checked:1');
+			const label = document.getElementById(&quot;cb-label-md-checkbox-1&quot;);
+			label.classList.remove(this.checked ? 'checkbox-label-unchecked' : 'checkbox-label-checked');
+			label.classList.add(this.checked ? 'checkbox-label-checked' : 'checkbox-label-unchecked');
+		} catch (error) {
+			console.warn('Checkbox checked:1 error', error);
+		}
+		return true;
+	" checked="checked"><label id="cb-label-md-checkbox-1" for="md-checkbox-1" class="checkbox-label-checked">This is a checked checkbox.</label></div>
+<ul>
+<li class="md-checkbox joplin-checkbox"><div class="checkbox-wrapper"><input type="checkbox" id="md-checkbox-2" onclick="
+		try {
+			if (this.checked) {
+				this.setAttribute('checked', 'checked');
+			} else {
+				this.removeAttribute('checked');
+			}
+
+			postMessage('checkboxclick:unchecked:2');
+			const label = document.getElementById(&quot;cb-label-md-checkbox-2&quot;);
+			label.classList.remove(this.checked ? 'checkbox-label-unchecked' : 'checkbox-label-checked');
+			label.classList.add(this.checked ? 'checkbox-label-checked' : 'checkbox-label-unchecked');
+		} catch (error) {
+			console.warn('Checkbox unchecked:2 error', error);
+		}
+		return true;
+	"><label id="cb-label-md-checkbox-2" for="md-checkbox-2" class="checkbox-label-unchecked">This is a nested checkbox.</label></div></li>
+<li class="md-checkbox joplin-checkbox"><div class="checkbox-wrapper"><input type="checkbox" id="md-checkbox-3" onclick="
+		try {
+			if (this.checked) {
+				this.setAttribute('checked', 'checked');
+			} else {
+				this.removeAttribute('checked');
+			}
+
+			postMessage('checkboxclick:checked:3');
+			const label = document.getElementById(&quot;cb-label-md-checkbox-3&quot;);
+			label.classList.remove(this.checked ? 'checkbox-label-unchecked' : 'checkbox-label-checked');
+			label.classList.add(this.checked ? 'checkbox-label-checked' : 'checkbox-label-unchecked');
+		} catch (error) {
+			console.warn('Checkbox checked:3 error', error);
+		}
+		return true;
+	" checked="checked"><label id="cb-label-md-checkbox-3" for="md-checkbox-3" class="checkbox-label-checked">They can be checked too.</label></div></li>
+</ul>
+</li>
+</ul>
```

Comment:
With all the JS code this test could break easily and will be hard to fix. Could you maybe use the `checkboxRenderingType = 2` option to generate the HTML code? or somehow write the test so that it ignores the irrelevant bits?

Reply:
Thanks for the tip; I changed the tests to use `checkboxRenderingType =2`. The order of the tests doesn't matter anymore either.